### PR TITLE
[AAE-5280] Better error handling for user task

### PR DESCRIPTION
**Pull request by dhrn**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://alfresco.atlassian.net/browse/AAE-5280

**What is the new behaviour?**

fixed

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by eromano — APPROVED** [review]

File: lib/process-services-cloud/src/lib/task/directives/claim-task-cloud.directive.spec.ts [hunk 1]
```diff
@@ -37,6 +37,10 @@ describe('ClaimTaskCloudDirective', () => {
 
         @ViewChild(ClaimTaskCloudDirective, { static: false })
         claimTaskDirective: ClaimTaskCloudDirective;
+
+        onError(error: Error) {
```
Comment (DenysVuika) [inline lib/process-services-cloud/src/lib/task/directives/claim-task-cloud.directive.spec.ts:41]: this method does not do anything except returning what you give it, why it's needed?

File: lib/process-services-cloud/src/lib/task/directives/complete-task.directive.spec.ts [hunk 2]
```diff
@@ -41,6 +41,10 @@ describe('CompleteTaskDirective', () => {
         onCompleteTask(event: any) {
             return event;
         }
+
+        onError(error: Error) {
```
Comment (DenysVuika) [inline lib/process-services-cloud/src/lib/task/directives/complete-task.directive.spec.ts:45]: does not do anything with the `error` parameter

File: lib/process-services-cloud/src/lib/task/directives/unclaim-task-cloud.directive.spec.ts [hunk 3]
```diff
@@ -37,6 +37,10 @@ describe('UnClaimTaskCloudDirective', () => {
 
         @ContentChildren(UnClaimTaskCloudDirective)
         unclaimTaskDirective: UnClaimTaskCloudDirective;
+
+        onError(error: Error) {
```
Comment (DenysVuika) [inline lib/process-services-cloud/src/lib/task/directives/unclaim-task-cloud.directive.spec.ts:41]: same as above

**Review by DenysVuika — CHANGES_REQUESTED** [review]

File: lib/process-services-cloud/src/lib/task/directives/claim-task-cloud.directive.spec.ts (hunk 1 above)
Comment (dhrn) [reply lib/process-services-cloud/src/lib/task/directives/claim-task-cloud.directive.spec.ts:41]: we are checking the `error`  method is called or not when API failed.

**Review by DenysVuika — APPROVED** [review]

**Pull request merged**